boards/arm/nrf52_sparkfun/Kconfig.defconfig

```diff
@@ -0,0 +1,35 @@
+# Kconfig - Sparkfun nRF52832 breakout  board configuration
+#
+# Copyright (c) 2016 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52_SPARKFUN
+
+config BOARD
+	default nrf52_sparkfun
+
+if GPIO_NRF5
+
+config GPIO_NRF5_P0
+	default y
+
+endif # GPIO_NRF5
+
+if UART_NRF5
+
+config UART_NRF5_GPIO_TX_PIN
+	default 27
+
+config UART_NRF5_GPIO_RX_PIN
+	default 26
+
+config UART_NRF5_GPIO_RTS_PIN
```

Comment:
No need to set defaults pins for RTS and CTS if h/w flow control is default n.

Reply:
And 0 for both is wrong configuration for sure.